README.md

```diff
@@ -1,480 +1,133 @@
+partly because users need to manually rewrite copy-pasted code fragments.
+By combining a standardized and versioned set of extensions
+with some packaging standards, we can
+make it easy to publish and automatic to use WGSL libraries.
+A standard community library format is key to enable an WGSL library ecosystem.
 
-When writing bigger WGSL shaders, one tends to **split the code into reusable pieces**. Examples are re-using a set of lighting calculation functions and sharing a struct between two compute shaders and.
+Join us on [github](https://github.com/wgsl-tooling-wg/wgsl-import-spec)
+or [discord](https://discord.gg/FXhZDV8V) to help.
 
-However, current WGSL tooling is not built for this. The official language purposefully does not include this feature, nor does it provide adjacent features like namespaces, and there is no standardized extension yet.
+## WGSL Enhancements
 
-One important aspect is getting buy-in from **WGSL language servers**, as shader development with IDEs should be well supported.
+We're aiming to prioritize WGSL enhancements that are: 1) important for community projects,
+2) feel natural to the WGSL programmer,
+and 3) reasonable to integrate into tools.
 
-We also should account for **importing shader from libraries**. Ideally, users could upload WGSL shaders to existing package managers, which other users could then consume.
+A WGSL enhancement for `import` statements
+to enable importing WGSL elements like functions from other files is our first priority.
+See spec draft here: **[imports](./Imports.md)**.
 
-Finally, we want **multiple tools** which can compile WGSL-with-imports down to raw WGSL. Using WGSL-with-imports both in Rust projects and in web projects should be possible.
+Several other WGSL enhancements
+are on the [roadmap](#wgsl-enhancements-roadmap).
 
-## Guide-level explanation
+## Tools
 
-This variant is semantically most similar to the Typescript syntax, but comes with a recursive grammar that makes importing multiple importable items less verbose. The grammar is also heavily inspired by [Gleam](https://gleam.run/).
+We'll build and promote tools to support community standard WGSL.
 
-By placing an import at the very top of a file, one can either import an entire module module, or only specific importable items, such as functions, structs or types.
+**Linking** - Tools to bundle wgsl files together
+while translating extended WGSL into vanilla W3C WGSL.
 
-```
-// Importing a single item using a relative path
-import ./lighting/pbr;
+**Test Cases** - A collection of examples to help tools interoperate.
 
-// Importing multiple items
-import my/geom/sphere/{ draw, default_radius as foobar };
+**WGSL Language Servers** - Enable syntax directed editing in IDEs.
+Shader development with IDEs should be well supported.
 
-// Imports a whole module. Use it with `bevy_ui.name`
-import bevy_ui/*;
-```
+**Syntax Highlighters** - Display pretty extended WGSL on web pages with
```

Comment:
People will also be interested in a tree-sitter grammar and/or a kate grammar for IDEs like vscode.

Reply:
yah, good one. adding item